dataland-frontend/src/components/general/BulkDataResponseDialog.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div data-test="bulkDataResponseDialog">
+    <p>Edit or remove them.</p>
```

Comment:
Is the dialog used yet?
Is this the dialog?
![image](https://github.com/d-fine/Dataland/assets/104447806/b5abf3e4-a682-4dd2-8612-df72719b75cc)
If so, is the backend already able to provide the necessary information?

Reply:
No, it is ready for use, but backend is not ready. I have turned it off for now.
Once backend is prepared in phase 2, then it will be just a matter of simple request to check identifiers and if there are any rejected identifiers then we should open this dialog. If no rejected identifiers then we don't open the dialog, but we will just send the second request.

Reply:
Ok, can we move it to a diffreent branch and not have it in this PR at all? Because otherwise we would commit unnused code to main, which we should avoid if possible.

Reply:
Moved to branch `DALA-3177_extend_data_requests_PHASE_2`
Description of work here: https://github.com/d-fine/DatalandInternal/wiki/DALA%E2%80%903177-%E2%80%90-Phase-2-%E2%80%90-Identifiers-Request-Dialog

Reply:
ok, thank you